### ISSUE TITLE
Incorrect weapon q20 prediction

### ISSUE THREAD

**Issue by Pachuriko** [PoE-TradeMacro/POE-TradeMacro#289]
![q0](https://cloud.githubusercontent.com/assets/26472745/24024815/41ea088e-0ac7-11e7-8d8b-c9aa44a40ef8.png)

![q20](https://cloud.githubusercontent.com/assets/26472745/24024827/59e0c536-0ac7-11e7-9293-fd3912707f6a.png)

**Comment by Eruyome** [comment]
That's actually an ItemInfo issue, there's a different repository for it.
But it's ok since @aRTy42 also watches the TradeMacro issues.

**Comment by aRTy42** [comment]
It is a rounding issue that only happens when ItemInfo is included and launched via the TradeMacro.

I've spent 2 hours trying to figure out what went wrong and finally realised that the ItemInfo does not calculate it wrong. ItemInfo alone is fine and the copy which is distributed with the TradeMacro is fine aswell. I've found a solution for the format setting and formula to make it work, but the underlying issue might reach further, so I'll discuss it with Eruyome tomorrow.
